system-tests/tests/smoke/cre/workflow_don_load_test.go

```diff
@@ -0,0 +1,915 @@
+	}
+
+	setupOutput := setupTestEnvironment(t, testLogger, &in.TestConfig, nil, mustSetCapabilitiesFn, createCustomJobsFunc, WorkflowDONLoadTestCapanilitiesFactoryFn)
+
+	ctx := tests.Context(t)
+	// Log extra information that might help debugging
+	t.Cleanup(func() {
+		if t.Failed() {
+			logTestInfo(testLogger, in.WorkflowConfig.FeedID, in.WorkflowConfig.WorkflowName, setupOutput.feedsConsumerAddress.Hex(), setupOutput.forwarderAddress.Hex())
+
+			logDir := fmt.Sprintf("%s-%s", framework.DefaultCTFLogsDir, t.Name())
+
+			removeErr := os.RemoveAll(logDir)
+			if removeErr != nil {
+				testLogger.Error().Err(removeErr).Msg("failed to remove log directory")
+				return
+			}
+
+			_, saveErr := framework.SaveContainerLogs(logDir)
+			if saveErr != nil {
+				testLogger.Error().Err(saveErr).Msg("failed to save container logs")
+				return
+			}
+
+			debugDons := make([]*keystonetypes.DebugDon, 0, len(setupOutput.donTopology.DonsWithMetadata))
+			for i, donWithMetadata := range setupOutput.donTopology.DonsWithMetadata {
+				containerNames := make([]string, 0, len(donWithMetadata.NodesMetadata))
+				for _, output := range setupOutput.nodeOutput[i].Output.CLNodes {
+					containerNames = append(containerNames, output.Node.ContainerName)
+				}
+				debugDons = append(debugDons, &keystonetypes.DebugDon{
+					NodesMetadata:  donWithMetadata.NodesMetadata,
+					Flags:          donWithMetadata.Flags,
+					ContainerNames: containerNames,
+				})
+			}
+
+			debugInput := keystonetypes.DebugInput{
+				DebugDons:        debugDons,
+				BlockchainOutput: setupOutput.blockchainOutput,
+			}
+			lidebug.PrintTestDebug(t.Name(), testLogger, debugInput)
+		}
+	})
+
+	require.NoError(t, saveFeedAddresses(feedsAddresses))
+
+	//Get ocr key
+	kb := make([]ocr2key.KeyBundle, 0)
+	for _, don := range setupOutput.donTopology.DonsWithMetadata {
+		if flags.HasFlag(don.Flags, keystonetypes.MockCapability) {
+			for i, n := range don.DON.Nodes {
+				if i == 0 {
+					continue // Skip bootstrap nodeker
+				}
+
+				key, err := n.ExportOCR2Keys(n.Ocr2KeyBundleID) //TODO: @george-dorin FIX ME!!!
+				if err == nil {
+					b, err2 := json.Marshal(key)
+					require.NoError(t, err2)
+					kk, err2 := ocr2key.FromEncryptedJSON(b, nodeclient.ChainlinkKeyPassword)
+					require.NoError(t, err2)
+					kb = append(kb, kk)
+				} else {
+					testLogger.Error().Msgf("Could not export OCR2 key: %s", err)
+				}
+
+			}
+		}
+	}
+
+	// Export key bundles so we can import them later in another test, used when crib cluster is already setup and we just want to connect to mocks for a different test
+	require.NoError(t, saveKeyBundles(kb))
+
+	testLogger.Info().Msg("Connecting to mock capabilities...")
+
+	mocksClient := mock_capability.NewMockCapabilityController(testLogger)
+	mockClientsAddress := make([]string, 0)
+	if in.Infra.InfraType == "docker" {
+		mockClientsAddress = []string{"127.0.0.1:13401", "127.0.0.1:13402", "127.0.0.1:13403", "127.0.0.1:13404"}
```

Comment:
maybe add a comment that they are hardcoded? but what will happen if we add 1 more node to the DON? any chance to make this dynamic?